### PR TITLE
Add filter intervals to docs

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by jigold** [comment]
@catoverdrive see this pr.

**Comment by jigold** [comment]
@catoverdrive is working on this along with `grep`.

**Comment by tpoterba** [comment]
grep should return a list of strings!

**Comment by tpoterba** [comment]
it's still searchable.

File: python/hail/docs/methods/index.rst [hunk 1]
```diff
@@ -46,6 +46,7 @@ Methods
 
 .. autosummary::
 
+    filter_intervals
```
Comment (jigold) [inline python/hail/docs/methods/index.rst]: Can you put this in alphabetical order? Same with instances above.

**Pull request merged**